handlers/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: enable and restart httpd on RedHat/CentOS 7
+  service: name=httpd enabled=yes state=restarted
+  when:
+   - ansible_os_family == "RedHat"
+   - ansible_distribution_major_version == "7"
```

Comment:
this should work on 7 and 8, no major_version needed

Reply:
done